addons/ui/org.openhab.ui.basic/META-INF/MANIFEST.MF

```diff
@@ -16,7 +16,6 @@ Import-Package:
  javax.servlet.http,
  org.apache.commons.io,
  org.apache.commons.lang,
- org.apache.commons.net.util,

```

Comment:
Maybe it's just my machine :wink: , but with a local build I think this change results the following exceptions for me:

```
11:44:21.694 [INFO ] [smarthome.event.ExtensionEvent       ] - Extension 'package-standard' has been installed.
org.apache.felix.resolver.reason.ReasonException: Unable to resolve root: missing requirement [root] osgi.identity; osgi.identity=openhab-ui-basic; type=karaf.feature; version="[2.5.0.SNAPSHOT,2.5.0.SNAPSHOT]"; filter:="(&(osgi.identity=openhab-ui-basic)(type=karaf.feature)(version>=2.5.0.SNAPSHOT)(version<=2.5.0.SNAPSHOT))" [caused by: Unable to resolve openhab-ui-basic/2.5.0.SNAPSHOT: missing requirement [openhab-ui-basic/2.5.0.SNAPSHOT] osgi.identity; osgi.identity=org.openhab.ui.basic; type=osgi.bundle; version="[2.5.0.201901231358,2.5.0.201901231358]"; resolution:=mandatory [caused by: Unable to resolve org.openhab.ui.basic/2.5.0.201901231358: missing requirement [org.openhab.ui.basic/2.5.0.201901231358] osgi.wiring.package; filter:="(osgi.wiring.package=org.apache.commons.net.util)"]]
	at org.apache.felix.resolver.Candidates$MissingRequirementError.toException(Candidates.java:1343)
	at org.apache.felix.resolver.ResolverImpl.doResolve(ResolverImpl.java:392)
	at org.apache.felix.resolver.ResolverImpl.resolve(ResolverImpl.java:378)
	at org.apache.felix.resolver.ResolverImpl.resolve(ResolverImpl.java:332)
	at org.apache.karaf.features.internal.region.SubsystemResolver.resolve(SubsystemResolver.java:257)
	at org.apache.karaf.features.internal.service.Deployer.deploy(Deployer.java:388)
	at org.apache.karaf.features.internal.service.FeaturesServiceImpl.doProvision(FeaturesServiceImpl.java:1025)
	at org.apache.karaf.features.internal.service.FeaturesServiceImpl.lambda$doProvisionInThread$13(FeaturesServiceImpl.java:964)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: org.apache.felix.resolver.reason.ReasonException: Unable to resolve openhab-ui-basic/2.5.0.SNAPSHOT: missing requirement [openhab-ui-basic/2.5.0.SNAPSHOT] osgi.identity; osgi.identity=org.openhab.ui.basic; type=osgi.bundle; version="[2.5.0.201901231358,2.5.0.201901231358]"; resolution:=mandatory [caused by: Unable to resolve org.openhab.ui.basic/2.5.0.201901231358: missing requirement [org.openhab.ui.basic/2.5.0.201901231358] osgi.wiring.package; filter:="(osgi.wiring.package=org.apache.commons.net.util)"]
	at org.apache.felix.resolver.Candidates$MissingRequirementError.toException(Candidates.java:1343)
	... 12 more
Caused by: org.apache.felix.resolver.reason.ReasonException: Unable to resolve org.openhab.ui.basic/2.5.0.201901231358: missing requirement [org.openhab.ui.basic/2.5.0.201901231358] osgi.wiring.package; filter:="(osgi.wiring.package=org.apache.commons.net.util)"
	at org.apache.felix.resolver.Candidates$MissingRequirementError.toException(Candidates.java:1343)
	... 13 more
11:44:30.928 [ERROR] [.core.karaf.internal.FeatureInstaller] - Failed installing 'openhab-ui-homebuilder, openhab-ui-basic, openhab-ui-habpanel, openhab-ui-paper': Unable to resolve root: missing requirement [root] osgi.identity; osgi.identity=openhab-ui-basic; type=karaf.feature; version="[2.5.0.SNAPSHOT,2.5.0.SNAPSHOT]"; filter:="(&(osgi.identity=openhab-ui-basic)(type=karaf.feature)(version>=2.5.0.SNAPSHOT)(version<=2.5.0.SNAPSHOT))" [caused by: Unable to resolve openhab-ui-basic/2.5.0.SNAPSHOT: missing requirement [openhab-ui-basic/2.5.0.SNAPSHOT] osgi.identity; osgi.identity=org.openhab.ui.basic; type=osgi.bundle; version="[2.5.0.201901231358,2.5.0.201901231358]"; resolution:=mandatory [caused by: Unable to resolve org.openhab.ui.basic/2.5.0.201901231358: missing requirement [org.openhab.ui.basic/2.5.0.201901231358] osgi.wiring.package; filter:="(osgi.wiring.package=org.apache.commons.net.util)"]]
```

Is there some underlying other issue that causes Commons Net not to be installed? 

We have similar issues with:

* https://github.com/openhab/openhab2-addons/issues/4709
* https://github.com/openhab/openhab2-addons/issues/4574

Reply:
This is exactly the exception that this change should prevent - and it worked for me locally.
I'd assume that your local build still used an older artifact of the Basic UI then.

Reply:
I had forgotten to use the .kar file so it was downloading the UIs (without your fixes) from the online repo. :relieved: